### PR TITLE
chore(workflows): add CI/CD workflows

### PR DESCRIPTION
**Pull request by rinaok**

### CODE REVIEW

File: .github/workflows/pull-request-main.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  build:
+    timeout-minutes: 15
+    if: github.event.pull_request.draft == false
+    name: Build
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - run: printf "registry=https://npm.pkg.github.com/Vonage\n_authToken=\${{ secrets.NPM_VVD_VNG_AUTOMATION_TOKEN }}\n//npm.pkg.github.com/:_authToken=\${{ secrets.NPM_VVD_VNG_AUTOMATION_TOKEN }}\nalways-auth=true" > .npmrc
```
Comment (yinonov) [inline .github/workflows/pull-request-main.yml:28]: can we dry-run this? we dont want to deploy anything not meaningful at this point of exploration

**Review by yinonov — CHANGES_REQUESTED** [review]

File: .github/workflows/pull-request-main.yml (hunk 1 above)
Comment (rinaok) [reply .github/workflows/pull-request-main.yml:28]: It's not for deployment

Comment (yinonov) [reply .github/workflows/pull-request-main.yml:28]: ok. just makes sure later this publishes a tagged version and not the semver

File: .github/workflows/pull-request.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  build:
+    timeout-minutes: 15
+    # if: github.event.pull_request.draft == false
+    name: Build
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+      - run: printf "registry=https://npm.pkg.github.com/Vonage\n_authToken=\${{ secrets.NPM_VVD_VNG_AUTOMATION_TOKEN }}\n//npm.pkg.github.com/:_authToken=\${{ secrets.NPM_VVD_VNG_AUTOMATION_TOKEN }}\nalways-auth=true" > .npmrc
```
Comment (yinonov) [inline .github/workflows/pull-request.yml:28]: still required?

Comment (rinaok) [reply .github/workflows/pull-request.yml]: no

**Pull request closed without merging**